app/views/debit_direct_files/index.html.haml

```diff
@@ -0,0 +1,19 @@
+= contextual_links
+
+= boot_page_title
+
+%table.table.table-striped.people.collection
+  %thead
+    %tr
+      %th= t_attr :id, DebitDirectFile
```

Comment:
If you use `t_attr` and do not specify the class, it will magically use the controller to look it up;-)